addons/website_event_crm/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Website Events CRM',
+    'version': '1.0',
+    'category': 'Website/Website',
+    'website': 'https://www.odoo.com/page/events',
+    'description': "Add access to different types of lead creation",
```

Comment:
it's not access but more display (access refers to access rules or access rights)